### PR TITLE
Added rangy@1.2.3

### PR DESCRIPTION
**Pull request by bargosc**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -195,6 +195,7 @@
   "raphael": "github:DmitryBaranovskiy/raphael",
   "ratchet": "github:maker/ratchet",
   "raven-js": "github:getsentry/raven-js",
+  "rangy": "github:timdown/rangy-release",
```
Comment (guybedford) [inline registry.json]: Trying to keep the registry alphabetical if possible.

File: package-overrides/github/timdown/rangy-release@1.2.3.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "format": "global",
+  "main": "rangy-core",
+
```
Comment (guybedford) [inline package-overrides/github/timdown/rangy-release@1.2.3.json]: Blank line

File: package-overrides/github/timdown/rangy-release@1.2.3.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "shim": {
+    "rangy-core": {
+      "exports": "rangy"
+    },
+    "rangy-cssclassapplier": {
+      "deps": ["rangy"]
```
Comment (guybedford) [inline package-overrides/github/timdown/rangy-release@1.2.3.json]: Do you mean `deps: ['./rangy-core']`?

**Comment by bargosc** [comment]
fixed

**Pull request merged**